src/installation/guides/index.md

```diff
@@ -3,4 +3,5 @@
 Got weird hardware or an unusual setup? This is the section for guides about how
 to install Void on things that it doesn't usually like.
 
-[Installing Void with Full Disk Encryption](fde.md)
+- [Installing Void via chroot (x86 or x86_64)](chroot.md)
+- [Installing Void with Full Disk Encryption](fde.md)
```

Comment:
Oh, could you add a `## Section Contents` header, please?

Reply:
yes